mkdocs.yml

```
@@ -43,6 +43,9 @@ nav:
     - Add your extensions: 'extensions/custom.md'
     - S3 Storage: 'extensions/s3.md'
   - Deploy:
+    - How can I deploy it?: 'deployment/index.md'
+    - System components: 'deployment/services.md'
+    - Configuration:  'deployment/configuration.md'
     - Kubernetes: 'deployment/kubernetes.md'
     - OpenShift: 'deployment/openshift.md'
```

How about the following order to get to the "higher-level" pages first (how to deploy on platforms) and then the lower-level pages after (the configuration definitions)? Like we do for most packages, where we place the API at the end. Especially since System components is empty right now.
 
```suggestion
    - How can I deploy it?: 'deployment/index.md'
    - OpenShift: 'deployment/openshift.md'
    - Kubernetes: 'deployment/kubernetes.md'
    - System components: 'deployment/services.md'
    - Configuration:  'deployment/configuration.md'
```